### PR TITLE
Fixes view history tab auto collapse

### PR DESCRIPTION
**Pull request by vkghub**
[ME-179400163](https://www.pivotaltracker.com/n/projects/2502930/stories/179400163)

### CODE REVIEW

**Comment by markgoho** [comment]
@vkghub if the PR title includes the word fix, there absolutely must either be:
1. updated/added specs 
2. some written explanation of why there's no change to specs

**Comment by vkghub** [comment]
> @vkghub if the PR title includes the word fix, there absolutely must either be:
> 
> 1. updated/added specs
> 2. some written explanation of why there's no change to specs

Totally missed it, pushed a commit for specs.

File: features/step_definitions/verification_process_steps.rb [hunk 1]
```diff
@@ -121,6 +121,10 @@ def user(*traits)
   find(:xpath, IvlDocumentsPage.view_history_option).click
 end
 
+And(/^Individual clicks on verify$/) do
+  find(:xpath, IvlDocumentsPage.verify_option).click
```
Comment (markgoho) [inline features/step_definitions/verification_process_steps.rb:125]: we should not be using xpath at all anymore

File: features/support/object_model_pages/ivl/homepage/ivl_documents_page.rb [hunk 2]
```diff
@@ -22,4 +22,8 @@ def self.local_mec_evidence_actions
   def self.view_history_option
     "//div[@class='selectric-scroll']/ul/li[contains(text(), 'View History')]"
   end
+
+  def self.verify_option
+    "//div[@class='selectric-scroll']/ul/li[contains(text(), 'Verify')]"
```
Comment (markgoho) [inline features/support/object_model_pages/ivl/homepage/ivl_documents_page.rb:27]: please use a page object model or css selector here

do not assert on hard-coded english text

**Review by markgoho — CHANGES_REQUESTED** [review]

File: features/step_definitions/verification_process_steps.rb (hunk 1 above)
Comment (vkghub) [reply features/step_definitions/verification_process_steps.rb]: fixed

File: features/support/object_model_pages/ivl/homepage/ivl_documents_page.rb (hunk 2 above)
Comment (vkghub) [reply features/support/object_model_pages/ivl/homepage/ivl_documents_page.rb]: removed xpath

File: features/step_definitions/verification_process_steps.rb [hunk 3]
```diff
@@ -121,6 +121,10 @@ def user(*traits)
   find(:xpath, IvlDocumentsPage.view_history_option).click
 end
 
+And(/^Individual clicks on verify$/) do
+  find('li', :text => "Verify").click
```
Comment (markgoho) [inline features/step_definitions/verification_process_steps.rb:125]: great, now we just need to replace the stuff inside find with a page object model

**Review by markgoho — CHANGES_REQUESTED** [review]

File: features/step_definitions/verification_process_steps.rb (hunk 3 above)
Comment (vkghub) [reply features/step_definitions/verification_process_steps.rb:125]: can you provide the reference code?

**Pull request closed without merging**